azure-servicebus/src/main/java/com/microsoft/azure/servicebus/ClientSettings.java

```diff
@@ -18,7 +18,12 @@
     private RetryPolicy retryPolicy;
     private Duration operationTimeout;
     private TransportType transportType;
-    
+
+    private String proxyHostName = null;
+    private int proxyHostPort = 0;
+    private String proxyUserName = null;
+    private String proxyPassword = null;
```

Comment:
can this be a separate class. and provide a constructor for it. don't expose set.

Reply:
Yes, I will work on making those changes.

Reply:
For now this is waiting on standardizing proxy support across SB & EH